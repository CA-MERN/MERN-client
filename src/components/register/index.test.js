import { shallow } from 'enzyme';
import React from 'react';

import Register from '.';

it('renders', () => {
    expect(shallow(<Register />)).toMatchSnapshot();
});

it('expects text to render', () => {
    expect(shallow(<Register />).text()).toMatch('First NameLast Name')
})
