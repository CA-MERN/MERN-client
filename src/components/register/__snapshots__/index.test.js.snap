// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<form
  onSubmit={[Function]}
>
  <label
    htmlFor="firstName"
  >
    First Name
  </label>
  <input
    id="firstName"
    name="firstName"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
    value=""
  />
  <label
    htmlFor="lastName"
  >
    Last Name
  </label>
  <input
    id="lastName"
    name="lastName"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
    value=""
  />
  <label
    htmlFor="email"
  >
    Email Address
  </label>
  <input
    id="email"
    name="email"
    onBlur={[Function]}
    onChange={[Function]}
    type="email"
    value=""
  />
  <label
    htmlFor="password"
  >
    Password
  </label>
  <input
    id="password"
    name="password"
    onBlur={[Function]}
    onChange={[Function]}
    type="password"
    value=""
  />
  <label
    htmlFor="confirmPassword"
  >
    Password
  </label>
  <input
    id="confirmPassword"
    name="confirmPassword"
    onBlur={[Function]}
    onChange={[Function]}
    type="password"
    value=""
  />
  <button
    type="submit"
  >
    Submit
  </button>
</form>
`;
